posthog/hogql_queries/ai/test/test_traces_query_runner.py

```diff
@@ -935,3 +935,44 @@ def test_aggregates_full_trace_events_with_property_filters(self):
         ).calculate()
         self.assertEqual(len(response.results), 1)
         self.assertEqual(response.results[0].inputTokens, 2)
+
+    def test_removes_duplicate_events(self):
```

Comment:
Question: Would this test actually validate the `arrayDistinct`, or would ClickHouse insert-time deduplication do the work instead? 🤔 Is it possible to bypass it somehow?

It's rather an observation, but would it add value to check passing with `arrayDistinct` removed? Because I'm afraid it could work 😅